lib/g_searcher_web/templates/report/show.html.eex

```diff
@@ -52,5 +52,5 @@
 </section>
 
 <section class="report-search-results">
-  <%= render "_search-result-table.html", search_results: @search_results, conn: @conn %>
+  <%= render GSearcherWeb.SharedView, "_search-result-table.html", assigns %>
```

Comment:
Does this work? so we pass only needed assign.
```suggestion
  <%= render GSearcherWeb.SharedView, "_search-result-table.html", search_results: @search_results %>
```

Reply:
Yes it does, this assigns is different from the `conn.assigns` attribute. This basically is passing everything that we've got from the controller, including the `@conn`. I think I forgot to change this back 😂 

We're using this way on Zipmex 😄

Reply:
@bterone I am not sure about the benefit to pass everything to the partial template. Maybe you can give me some insight into this case.

Anyway PR LGTM, approved 👍

Reply:
If you have time, you can share in #elixir-guild for other developers too 🙌